tasks/python-reference-task/README.md

```diff
@@ -0,0 +1,29 @@
+# @cumulus/python-reference-task
+
+This 'task' is a reference task that is included with Cumulus to allow integration testing of the `cumulus message adapter` with a built python lambda.
+
+## Use
+
+Developmental use of this lambda is intended to be simple - the task returns a static processing output, integration tests can be then built against the `Reference Task` step in the `PythonProcess` workflow.
```

Comment:
What is the `PythonProcess` workflow? It doesn't link to anywhere and I don't see anything with that name

Reply:
This needs fixing, the workflow was renamed after the task was written.

Reply:
should be `PythonReferenceWorkflow` , with link to source.